content/docs/1.8.0/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -291,6 +291,8 @@ Once the secret is created and Longhorn's settings are saved, navigate to the ba
 
 If you don't get any error messages, try creating a backup and confirm the content is pushed out to your new bucket.
 
+If the Backup Target page displays an error string of just the word 'error', you may learn more by checking the JSON response of the page's internal fetch of `/v1/backuptargets`. Note that in this response, errors from GCP will be labeled "AWS Error", e.g. "AWS Error:  AccessDenied". (See [this issue](https://github.com/longhorn/longhorn/issues/10428)).
```

Comment:
Will we have an improvement to show the real error message on the GUI, for example, hovering the `Error` in a row on the `Backup Target` page?

Reply:
Sure, we need as discussed before. Could you add the link of the improvement ticket?

Reply:
https://github.com/longhorn/longhorn/issues/10428